include/swupd-build-variant.h

```diff
@@ -25,6 +34,7 @@
 #define TAR_PERM_ATTR_ARGS "--preserve-permissions " TAR_XATTR_ARGS
 #else /* SWUPD_WITHOUT_XATTRS */
 #define TAR_PERM_ATTR_ARGS "--preserve-permissions "
-#endif
+
+#endif /* SWUPD_WITH_BSDTAR */
```

Comment:
Duh. Still not right. I was expecting the #ifdef SWUPD_WITH_XATTRS to be inside the section for GNU tar and hence placed my endif comment where it is now, which (after actually reading what it does) of course isn't right.

Third time is the charm - force-pushed.